packages/react-components/icons/RadioButton.tsx

```diff
@@ -0,0 +1,50 @@
+import colors from '@celo/react-components/styles/colors'
+import * as React from 'react'
+import { StyleSheet, View } from 'react-native'
+import Svg, { Circle } from 'svgs'
+
+
+interface Props {
+  height?: number
+  color?: string
+  selected?: boolean
+  disabled?:boolean
+}
+
+const styles = StyleSheet.create({
+  circle: {
+    paddingTop: 2,
+    height: 20,
+    width: 20,
+    borderRadius: 12,
+    borderWidth: 2,
+    borderColor: colors.gray4,
+  },
+})
+
+export default class RadioButton extends React.PureComponent<Props> {
+  static defaultProps = {
+    width: 20,
+    height: 20,
+    color: colors.greenUI,
+    selected: true, 
+    disabled: false,
+  }
+  
+  
+
+  render() {
+    return (
+      <Svg
+        xmlns="http://www.w3.org/2000/svg"
+        height={this.props.height}
+        width={this.props.height}
+        viewBox="0 0 20 20"
+      >
+      <Circle cx="10" cy="10" r="9" stroke={this.props.disabled ? colors.gray2 : this.props.color} strokeWidth={2} />  
+      {this.props.selected ? <Circle cx="10" cy="10" r="6" fill={this.props.color} /> : <View style={styles.circle} />
```

Comment:
Could you do something even more simple here and just have a conditional for the `fill` property? Something like: `fill = {this.props.selected ? this.props.color : undefined}`

Reply:
I think `stroke` would have to change too actually...but still think that's more simple than rendering a `View` circle with all it's own styling and sizing

Reply:
mm sure, i can try something out! that makes sense

Reply:
i think it's actually a little simpler to keep it as is, since we are also keeping track of whether or not something is `disabled` for the stroke. It gets a little complicated since the stroke also changes if the RadioButton is disabled haha. Do you have something specific in mind?

Reply:
The following are not crucial by any means, I just believe these things help with readability and robustness:

- Defining things only once and reusing (color variable):

On `RadioIcon` we had this line instead: `{selected && <Circle cx="10" cy="10" r="6" fill={color} />}`. `color` was defined as `const color = selected ? colors.greenUI : colors.gray3` which in this case could be changed to `const color = selected ? props.color : colors.gray3` to achieve the same result.

I did notice that the outer circle is using `colors.gray2` but the inner use is using `colors.gray4`. It would probably be best to unify those to be the same. Defining the color only once would prevent this issue.

- Having the UI in one place:

This doesn't always apply, but in this case, if we have everything defined in the `render` method there's no need to scroll up to look at the `styles.circle` to understand the disabled state, one can understand the whole thing by looking at only one place. Also, I'm not sure why the disabled state is necessary, isn't it doing the same thing as the first `Circle`?

- The last thing is that we're using standard css (`styles.circle`) instead of properties from the `svgs` package, which I'm not completely sure if it has any implications in certain phone sizes (this might not be an issue).


Again, this code is working, so great job on that and the comments above might seem a bit opinionated but I do believe that they lead easier to read and maintain code.

Reply:
i think the disabled state was just from the design, and they are different colors, and without the second circle in the middle. 

I updated the line in the latest commit to be: `const fill = (this.props.selected && !this.props.disabled) ? this.props.color : undefined;` 
since we're keeping track that it's both not selected and not disabled to have the middle circle.

Reply:
i also removed the standard css usage and simplified it to use the svgs package instead! thanks for your comments Gonza

Reply:
> think the disabled state was just from the design, and they are different colors

I see, I hadn't caught that :P